EVSL_1.0/SRC/landosG.c

```diff
@@ -188,15 +162,8 @@ int LanDosG(const int nvec, const int msteps, const int degB, int npts,
       if (ifGenEv) {
         /* znew = znew - Z(:,1:k)*V(:,1:k)'*znew */
         CGS_DGKS2(n, k, NGS_MAX, Z, V, znew, wk);
-        /* -------------- NOTE: B-matvec
-         *                vnew = B * znew */
-        if (degB <= 0) {
-          printf("LanDos with degB <= 0 not yet implemented");
-          exit(-1);
-        } else {
-          pnav(pol_sol.mu, pol_sol.deg, pol_sol.cc, pol_sol.dd, znew, vnew,
-               wk);  // Ish
-        }
```

Comment:
Thanks, I directly translated the Matlab code which did not contain solve_lt, solve_b. I'll keep this in mind for the future.